clip.py

```diff
@@ -123,6 +140,27 @@ def channels(self, clip):
         else:
             return self.data[clip.audio_file].shape[1]
 
+    @property
+    def outputs(self):
+        seen = set()
```

Comment:
Maybe I have miss something but this does same thing : 

    return list(set(c.output for c in self.clips))
and maybe we can return a set instead of list : 

    return set(c.output for c in self.clips)